test/backup_service/backup_service_test.go

```diff
@@ -60,6 +58,34 @@ var _ = Describe(
 					Expect(err).To(HaveOccurred())
 				})
 
+				It("Should fail when backup service version 2.0 image is given", func() {
```

Comment:
Should fail when backup service version is before 3.0.
This statement seems more generic.